procdesc.cc

```diff
@@ -109,27 +109,39 @@ void CheckChildFinished(pid_t pid, bool signaled=false) {
 
```

Comment:
BTW, there's a TODO near the top of this file (~l.30) that you can probably remove when you're done &ndash; thanks!

Reply:
There are still a few tests that use sleep, I'll try to fix those too and then remove the comment.

Reply:
I'll open a separate PR for those remainig tests though.